src/modules/ChannelList/utils.js

```diff
@@ -34,18 +34,6 @@ const createEventHandler = ({
         });
       }
     },
-    onUserReceivedInvitation: (channel, inviter, invitees) => {
```

Comment:
why this is removed?

Reply:
Because it's not used, but I will add it again. There was a customer request.